auth/templates/uniform/allauth/account/login.html

```diff
@@ -124,7 +124,20 @@
 <form class="login uniForm" method="POST" action="{% url 'account_login' %}">
  	{% csrf_token %}
   <fieldset class="inlineLabels">
-  {{ form|as_uni_form }}
```

Comment:
if _uni_form isn't working, we should remove it from the requirements.txt as well.  You may want to try this again now that we updated it though.